### PR TITLE
[bnxt] Use hexadecimal values in PCI_ROM entries

### PR DESCRIPTION
**Pull request by jw14812**
Use hexadecimal values instead of macros in PCI_ROM entries so Perl script
can parse them correctly. Move PCI_ROM entries from header file to C file.
Integrate bnxt_vf_nics array into PCI_ROM entries by introducing
BNXT_FLAG_PCI_VF flag into driver_data field. Add whitespaces in PCI_ROM
entries for style consistency.

### CODE REVIEW

File: src/drivers/net/bnxt/bnxt.h [hunk 1]
```diff
@@ -885,15 +886,19 @@ struct bnxt {
...
+#define DID_57454_MHB_RDMA   0x1605
+#define DID_57454_VF_RDMA    0x1606
+#define DID_57454_VF         0x1609
+#define DID_57454            0x1614
+#define DID_58802            0xD802
+#define DID_58804            0xD804
+#define DID_57454_VF_HV      0x1607
+#define DID_57454_VF_HV_RDMA 0x1608
+#define DID_57412_VF_HV      0x16c5
+#define DID_57412_VF_HV_RDMA 0x16bd
```
Comment (NiKiZe) [inline src/drivers/net/bnxt/bnxt.h:902]: With the new flag, these defines are unused, correct?

**Review by NiKiZe — COMMENTED** [review]
Continuation from #331, This looks cleaner

Any reason why the `/* Stratus Device IDs */` defines are added/keept?

File: src/drivers/net/bnxt/bnxt.h (hunk 1 above)
Comment (jw14812) [reply src/drivers/net/bnxt/bnxt.h]: Correct observation.  I have removed the unnecessary defines as suggested

**Review by NiKiZe — APPROVED** [review]

**Comment by NiKiZe** [comment]
@mcb30 I think that this is a good step in the right direction.

**Review by mcb30 — APPROVED** [review]

**Pull request merged**